src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -78,7 +78,23 @@ discoverable from other clusters in the Submariner deployment.
 |:-----------------------------|:----------------------------------------------------------------------------|
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default `$HOME/.kube/config`)
 | `--kubecontext` `<string>`   | Kubeconfig context to use
-| `--namespace` `<string>`     | Namespace in which the Service to be exported belongs
+| `--namespace` `<string>`     | Namespace to use
+
+If no `namespace` flag is specified, it uses the default namespace from the current context, if present, otherwise it uses `default`.
+
+### `unexport`
+
+#### `unexport service`
+
+`subctl unexport service [flags] <name>` stops exporting a resource so that it's no longer accessible to other clusters.
```

Comment:
```suggestion
`subctl unexport service [flags] <name>` stops exporting a resource so that it's no longer discoverable from other clusters.
```

to make it symmetric with the `export` documentation (the service is technically still accessible).

Reply:
Ah, yes, the documentation for `subctl export` doesn’t match the help text. But the help text describes `subctl export` in general, whereas this documentation is for `subctl unexport service` specifically; the corresponding help text for that is

> This command removes the ServiceExport resource with the given name which in turn stops the Service of the same name from being exported to other clusters

The documentation should either follow the pattern set for `subctl export service` in the existing text above, or match the actual help text for `subctl export service`.